lib/ui/platform_dispatcher.dart

```diff
@@ -185,6 +185,15 @@ class PlatformDispatcher {
   /// otherwise.
   FlutterView? view({required int id}) => _views[id];
 
+  /// The ID of the currently focused [FlutterView].
+  ///
+  /// Returns -1 if no view is focused.
+  ///
+  /// See also:
+  /// * [FlutterView.hasFocus] to ask a specific view if it is focused.
+  int get focusedViewId => _focusedViewId;
+  int _focusedViewId = -1;
```

Comment:
Interesting. Is this the convention around view ids? Is there a reason not to make this nullable?

Reply:
I have the same question :)